### DISCUSSION TITLE
Senec v3 Australia

### DISCUSSION THREAD

**Discussion by matheuscanela** [marq24/ha-senec-v3#89] [General]
Hey folks 

I am planning to buy a solar + battery system, and an Australian company offered senec v3, after checking on the Internet, I can that they are not operating on Australia anymore but honoring the warranty - 20 years which is well above anyone here, so not even sure if they will actually honor it

I was initially considering sungrow batteries as my idea is to fully manage the inverter, and sell or buy from the grid when it is cheaper, as here in Australia we have Amber which sells electricity using the whole sales price.

Would you mind helping me with the questions below

1. As a owner - Is the battery reliable?
2. Are you able to fully manage the inverter? Example, sell electricity to the grid from the battery, or charge from grid/solar just using the integration?

Not sure yet if Senec is a good option, but trying to figure out the ROI

Thanks so much

**Comment by marq24** [comment]
I am sorry - notifications for discussions had been off - I guess this is not "up2date" any longer?

To answer your questions from my personal/private experience (as v3 Owner for more then 2 years now)
1) No it is not - paid for 10kWh - during the past 2 years I could only use it for less then 9 month
2) No you can't manage anything "fully" - you can just request to LOAD the Batterry with power from the GRID - but you just can request to load it Full / you can't hold the current level / you can't discharge from Battery to grid
